### PR TITLE
Adding find_id method to collection

### PR DESCRIPTION
**Pull request by jmatth**
[PERL-518](https://jira.mongodb.org/browse/PERL-518)

### CODE REVIEW

File: lib/MongoDB/Collection.pm [hunk 1]
```diff
@@ -727,6 +727,48 @@ sub find_one {
     )->execute->next;
 }
 
+=method find_id
```
Comment (jmatth) [inline lib/MongoDB/Collection.pm]: How should I go about this documentation? Right now it's copied almost verbatim from the `find_one` documentation. Would it be ok to just say it takes an `id` and to look at the `find_one` docs for more info?

Comment (xdg) [inline lib/MongoDB/Collection.pm]: I think shorter documentation that references `find_one` would be fine.

Minor note: for a POD `C<...>` expression, if there is a `>` inside, you need to write it in a longer form like this (delimited with doubled brackets and spaces): `C<< ... >>`.

For example: `C<< _id => $id >>`

**Comment by xdg** [comment]
After you simplify the docs, it will LGTM.

**Comment by jmatth** [comment]
@xdg Cut out most of the duplicate documentation. Is there a way to link to the documentation for `find_one` instead of just mentioning it?

**Comment by xdg** [comment]
Yes: `L<find_one|/find_one>` (first part before pipe is the display text, second part is the link)

**Comment by jmatth** [comment]
Ok, should be good now.

**Pull request closed without merging**